scripts/weather.py

```diff
@@ -43,10 +42,10 @@
     if d['WT14'] == '1' or d['WT08'] == '1' or d['WT13'] == '1':
         d['WEATHER'] = 'drizzle'
 
-    if d['WT16'] == '1' or d['WT17'] == '1' or d['WT05'] == '1' or d['WT03'] == '1':
+    if d['WT16'] == '1' or d['WT17'] == '1' or d['WT05'] == '1' or int(d['PRCP']) > 0:
```

Comment:
What's the `int` for here? Can you make this code clearer by using floor/ceil or comparing to some other number?

Reply:
`d['PRCP']` here is a string. I was half-following the `d['PRCP'] = 1.0 * int(d['PRCP'])/10` code below.

Were you thinking it could be a float and we should `floor(float(d['PRCP'])) > 0.0`?

Reply:
Oh, thanks. Yeah, I didn't see that it was a string. If the number is never a float, then `int` is good.